src/main/java/net/minecraftforge/client/model/ModelLoader.java

```diff
@@ -384,6 +384,10 @@ public VanillaModelWrapper(ResourceLocation location, ModelBlock model, boolean
                 else
                 {
                     IModel parent = ModelLoaderRegistry.getModelOrLogError(model.getParentLocation(), "Could not load vanilla model parent '" + model.getParentLocation() + "' for '" + model);
+                    if (parent instanceof FancyMissingModel)
```

Comment:
Formatting Consistency 
```suggestion
                    if(parent instanceof FancyMissingModel)
```